web/src/main/kotlin/no/nav/su/se/bakover/web/routes/behandling/BehandlingRoutes.kt

```diff
@@ -212,7 +211,7 @@ internal fun Route.behandlingRoutes(
                             is KunneIkkeLageBrevutkast.FantIkkePerson -> NotFound.message("Fant ikke person")
                             is KunneIkkeLageBrevutkast.FikkIkkeHentetSaksbehandlerEllerAttestant -> InternalServerError.message("Klarte ikke hente informasjon om saksbehandler og/eller attestant")
                         }
-                        call.respond(resultat)
+                        call.svar(resultat)
```

Comment:
Hvis man brukte respond, så tolker frontenden det at alt gikk bra -> åpner ny tab for pdf -> og viser 400 bad request

Med svar så får frontenden den faktiske statusen backenden gir oss, og tilsvarende feilmelding

Reply:
call.svar kaller call.respondText 

```Responds to a client with a plain text response, using specified text
Params:
contentType - is an optional ContentType, default is ContentType.Text.Plain
status - is an optional HttpStatusCode, default is HttpStatusCode.OK```

Reply:
Kan se ut som Ktor gjør Resultat om til json? :grimacing: https://ktor.io/docs/responses.html#content-negotiation

Testene burde avdekket dette

Reply:
Dette var da litt interresant å se på😄